src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -4,7 +4,11 @@ import { connect } from 'react-redux';
 import LoadingIndicator from '@department-of-veterans-affairs/component-library/LoadingIndicator';
 import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
 
-import { isMultifactorEnabled, isVAPatient } from '~/platform/user/selectors';
+import {
+  isMultifactorEnabled,
+  isVAPatient,
+  isLOA3 as isLOA3Selector,
```

Comment:
Probably don't need to rename `isLOA3`, but looks good either way 👍

Reply:
Oh you're right. I did in my first pass and then I changed how I was using it.